File: Python/1000-1999/1040.py
def main():
    n1, n2, n3, n4 = map(float, input().split())

    m = (n1*2 + n2*3 + n3*4 + n4)/10

    print('Media: %.1f' %(m))
    
    if m >= 7:
        print('Aluno aprovado.')
    
    elif m < 6.9 and m >= 5.0:
        print('Aluno em exame.')
        ex = float(input())
        print('Nota do exame: %.1f' %(ex))

        m = (m+ex)/2

        if m >= 5:
            print('Aluno aprovado.')
            print('Media final: %.1f' %(m))
        
        else:
            print('Aluno reprovado.')

    else:
        print('Aluno reprovado.')


main()

File: C/1000-1999/1040.c
#include <stdio.h>

int main(void){
    float n1, n2, n3, n4, med;

    scanf("%f %f %f %f", &n1, &n2, &n3, &n4);
    med = (n1*2 + n2*3 + n3*4 + n4)/10;
    printf("Media: %.1f\n", med);

    if (med >= 7){
        printf("Aluno aprovado.\n");
    } else if (med >= 5){
        printf("Aluno em exame.\n");
        scanf("%f", &n1);
        printf("Nota do exame: %.1f\n", n1);
        med = (med+n1)/2;
        if (med >= 5){
            printf("Aluno aprovado.\n");

        } else{
            printf("Aluno reprovado.\n");
        }
        printf("Media final: %.1f\n", med);
    } else{
        printf("Aluno reprovado.\n");
    }

    return 0;
}
